packages/@aws-cdk/aws-backup/test/vault.test.ts

```diff
@@ -135,6 +136,89 @@ test('defaults to all notifications', () => {
   });
 });
 
+test('import from arn', () => {
+  // WHEN
+  let vaultArn = stack.formatArn({
+    service: 'backup',
+    resource: 'backup-vault',
+    resourceName: 'myVaultName',
+    sep: ':',
+  });
+  let vault = BackupVault.fromBackupVaultArn(stack, 'Vault', vaultArn);
+
+  // THEN
+  expect(vault.backupVaultName).toEqual('myVaultName');
+  expect(vault.backupVaultArn).toEqual(vaultArn);
+});
+
+test('import from name', () => {
+  // WHEN
+  let vaultName = 'myVaultName';
+  let vault = BackupVault.fromBackupVaultName(stack, 'Vault', vaultName);
+
+  // THEN
+  expect(vault.backupVaultName).toEqual(vaultName);
+  expect(vault.backupVaultArn).toEqual(stack.formatArn({
+    service: 'backup',
+    resource: 'backup-vault',
+    resourceName: 'myVaultName',
+    sep: ':',
+  }));
+});
+
+test('grant action', () => {
+  // GIVEN
+  let vaultName = 'myVaultName';
+  let vault = BackupVault.fromBackupVaultName(stack, 'Vault', vaultName);
+  let fn = new lambda.Function(stack, 'function', {
```

Comment:
I'd prefer using an IAM role here instead of using the relatively complex Lambda grantable